SPEC.md

```diff
@@ -865,9 +865,80 @@ task literals_paths {
     # does not exist, then `f2` is set to `None`.
     File? f2 = "/foo/bar.txt"
   }
+
+  # If baz.txt does not exist, this is an error.
```

Comment:
This is just making explicit the assertion above that bound declarations behave this way as well.